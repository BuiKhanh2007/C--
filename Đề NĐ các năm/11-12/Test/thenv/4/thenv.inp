6
1479721
2434657
5468961
5892426
2387562
4235423
