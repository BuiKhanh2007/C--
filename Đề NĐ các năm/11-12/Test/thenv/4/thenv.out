G
A
C
F
A
A
